TF introduced a change pinning versions to fixed major.minor or even patch versions
Loosen those a bit so we can build it with our versions.
See https://github.com/tensorflow/tensorflow/issues/44654

Author: Alexander Grund (TU Dresden)
Update v2.4.4: Alex Domingo (Vrije Universiteit Brussel)

--- tensorflow/tools/pip_package/setup.py.orig	2021-10-31 00:59:09.000000000 +0200
+++ tensorflow/tools/pip_package/setup.py	2022-02-08 10:36:43.677543730 +0100
@@ -77,18 +77,18 @@
 REQUIRED_PACKAGES = [
     'absl-py ~= 0.10',
     'astunparse ~= 1.6.3',
-    'flatbuffers ~= 1.12.0',
-    'google_pasta ~= 0.2',
+    'flatbuffers >= 1.12.0',
+    'google_pasta >= 0.2',
     'h5py ~= 2.10.0',
     'keras_preprocessing ~= 1.1.2',
-    'numpy ~= 1.19.2',
-    'opt_einsum ~= 3.3.0',
+    'numpy >= 1.16.0',
+    'opt_einsum >= 3.3.0',
     'protobuf >= 3.9.2',
-    'six ~= 1.15.0',
-    'termcolor ~= 1.1.0',
-    'typing_extensions ~= 3.7.4',
-    'wheel ~= 0.35',
-    'wrapt ~= 1.12.1',
+    'six >= 1.12.0',
+    'termcolor >= 1.1.0',
+    'typing_extensions >= 3.7.4.2',
+    'wheel >= 0.26',
+    'wrapt >= 1.12.1',
     # These packages needs to be pinned exactly as newer versions are
     # incompatible with the rest of the ecosystem
     'gast == 0.3.3',
@@ -118,7 +118,7 @@
 # BoringSSL support.
 # See https://github.com/tensorflow/tensorflow/issues/17882.
 if sys.byteorder == 'little':
-  REQUIRED_PACKAGES.append('grpcio ~= 1.32.0')
+  REQUIRED_PACKAGES.append('grpcio >= 1.32.0')
 
 
 # Packages which are only needed for testing code.
